desc/compute/_neoclassical.py

```diff
@@ -0,0 +1,509 @@
+    # τ is the bounce time, and I is defined in Nemov eq. 36.
+    return safediv(2.0, jnp.sqrt(jnp.abs(1 - pitch * B)))
+
+
+def _f1(grad_psi_norm_kappa_g, B, pitch, zeta):
+    return (
+        safediv(1 - 0.5 * pitch * B, jnp.sqrt(jnp.abs(1 - pitch * B)))
+        * grad_psi_norm_kappa_g
+        / B
+    )
+
+
+def _f2(B_r, B, pitch, zeta):
+    return safediv(1 - 0.5 * pitch * B, jnp.sqrt(jnp.abs(1 - pitch * B))) * B_r / B
+
+
+def _f3(K, B, pitch, zeta):
+    return jnp.sqrt(jnp.abs(1 - pitch * B)) * K / B
+
+
+@register_compute_fun(
+    name="Gamma_c",
+    label=(
+        # Γ_c = π/(8√2) ∫dλ 〈 ∑ⱼ [v τ γ_c²]ⱼ 〉
+        "\\Gamma_c = \\frac{\\pi}{8 \\sqrt{2}} "
+        "\\int d\\lambda \\langle \\sum_j (v \\tau \\gamma_c^2)_j \\rangle"
+    ),
+    units="~",
+    units_long="None",
+    description="Energetic ion confinement proxy, Nemov et al.",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=[
+        "min_tz |B|",
+        "max_tz |B|",
+        "B^phi",
+        "B^phi_r|v,p",
+        "|B|_r|v,p",
+        "b",
+        "grad(phi)",
+        "grad(psi)",
+        "|grad(psi)|",
+        "|grad(rho)|",
+        "|e_alpha|r,p|",
+        "kappa_g",
+        "iota_r",
+    ]
+    + Bounce2D.required_names,
+    resolution_requirement="z",  # FIXME: GitHub issue #1312. Should be "tz".
+    grid_requirement={"can_fft": True},
+    **_Bounce2D_doc,
+    quad2="Same as ``quad`` for the weak singular integrals in particular.",
+)
+@partial(
+    jit,
+    static_argnames=[
+        "Y_B",
+        "num_transit",
+        "num_quad",
+        "num_pitch",
+        "num_well",
+        "batch_size",
+        "spline",
+    ],
+)
+def _Gamma_c(params, transforms, profiles, data, **kwargs):
+    """Energetic ion confinement proxy as defined by Nemov et al.
+
+    Poloidal motion of trapped particle orbits in real-space coordinates.
+    V. V. Nemov, S. V. Kasilov, W. Kernbichler, G. O. Leitold.
+    Phys. Plasmas 1 May 2008; 15 (5): 052501.
+    https://doi.org/10.1063/1.2912456.
+    Equation 61.
+
+    The radial electric field has a negligible effect on alpha particle confinement,
+    so it is assumed to be zero.
+    """
+    # noqa: unused dependency
+    grid = transforms["grid"]
+
+    theta = kwargs["theta"]
+    Y_B = kwargs.get("Y_B", theta.shape[-1] * 2)
+    num_transit = kwargs.get("num_transit", 20)
+    num_pitch = kwargs.get("num_pitch", 64)
+    num_well = kwargs.get("num_well", Y_B * num_transit)
+    batch_size = kwargs.get("batch_size", None)
+    spline = kwargs.get("spline", True)
+    if "fieldline_quad" in kwargs:
+        fieldline_quad = kwargs["fieldline_quad"]
+    else:
+        fieldline_quad = leggauss(Y_B // 2)
+    if "quad" in kwargs:
+        quad = kwargs["quad"]
+    else:
+        quad = get_quadrature(
+            leggauss(kwargs.get("num_quad", 32)),
+            (automorphism_sin, grad_automorphism_sin),
+        )
```

Comment:
@ddudt last week the default quadrature for the 2D stuff here was using the quadratures best for weak singularities i.e.
```python
chebgauss2(num_quad)
```
and I have since corrected it.

The 1D stuff is unchanged since it was using the correct quadrature.